static/js/InstanceOverview.tsx

```diff
@@ -28,6 +34,47 @@ const InstanceOverview: FC<Props> = ({ instanceName }) => {
     queryFn: () => fetchInstance(instanceName),
   });
 
+  useEffect(() => {
+    if (instance) {
+      setControls(
+        <div className="u-instance-overview-controls">
```

Comment:
This is inline by default. So we can use an empty fragment here as a wrapper. And clean-up the `u-instance-overview-controls` class.

Reply:
Very strange. I tried that approach before first, and it wasn't working. It was causing buttons to be stacked vertically. That's why I had applied that style. Anyway, now that I tried again, it works as you said. Done @ [1ea1594](https://github.com/canonical/lxd-ui/commit/1ea159429bc0928d054872d87ec3c930dbe025d7)